demo/src/app/search-filters-consumer.component.ts

```diff
@@ -15,7 +15,8 @@ export class SearchFiltersConsumerComponent implements OnInit {
       states: [],
       events: [],
       case_fields: [],
-      jurisdiction: null
+      jurisdiction: null,
+     printableDocumentUrl: 'print-url'
```

Comment:
My only comment is to confirm correctness of or fix the singularity of this field.

Reply:
Done.   Resolved after making the field as optional.